themes/templates/themes/dark/articles/article-feature.html

```diff
@@ -0,0 +1,26 @@
+{% load static article_tags core_tags wagtailcore_tags wagtailembeds_tags wagtailimages_tags %}
+
+<div class="jumbotron main-feature overlay" style="height: 400px;">
+    {% image self.main_image fill-2048x1200-c100 as background_image %}
+    <div class="feature-image" style="background-image: url('{{ background_image.url }}')">
+        <div class="feature-image-overlay"
+             style="background: rgba( {% if self.image_overlay_color.rgb %} {{ self.image_overlay_color.rgb }} {% endif %},
+                     {% if self.opacity %} {{ self.opacity }} {% endif %})">
+        </div>
+    </div>
+    <div class="feature-wrapper">
+        <div class="feature-text">
+            {% include metadata %}
+        </div>
+    </div>
+
+</div>
+{% if not self.include_caption_in_footer %}
```

Comment:
This change should be made in the base template since it could be applied to any of the themes because the option is global.